x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/shared_components/form_row.scss

```
@@ -0,0 +1,3 @@
+.lnsInnerIndexPattern__inlineRow {
```

Is this the most appropriate class name for this element? Asking because the `lnsInnerIndexPattern` based classes seem to be found in the field list, not the configuration flyout. Based on the parent element classes, I'm wondering if something like the following would be better.

```suggestion
.lnsIndexPatternDimensionEditor__labelCustomRank {
```